Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/ICommandService.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+namespace Microsoft.MixedReality.Toolkit.Extensions.Experimental.SpectatorView
+{
+    public interface ICommandService
+    {
+        bool Register(string command, ICommandHandler handler);
```

Comment:
>Register [](start = 13, length = 8)

Nit: I'd name these RegisterCommandHandler / UnregisterCommandHandler so that they're a little more clear in context